packages/bpk-component-skeleton/src/BpkBaseSkeleton.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { cssModules } from '../../bpk-react-utils';
+
+import STYLES from './BpkBaseSkeleton.module.scss';
+
+const getClassName = cssModules(STYLES);
+
+type Props = {
+  className: string;
+  ariaLabel?: string;
+  styleObj?: {
+    width: string;
+    height: string;
+  };
+};
+
+const BpkBaseSkeleton = ({
+  ariaLabel,
+  className,
+  styleObj
+ }: Props) => {
+
+  const classNames = [getClassName('bpk-skeleton')];
+  if (className) {
+    classNames.push(className);
+  }
+
+  return (
+    <div role="status" aria-label={ariaLabel} className={classNames.join(' ')} style={styleObj}  />
```

Comment:
So I think `className={classNames.join(' ')}` can be `className={classNames}`

Reply:
Updated.